python/setup.py

```diff
@@ -4,39 +4,42 @@
 # SPDX-License-Identifier: MIT
 
 import os
+
 import setuptools
 
+
 def read_config_map():
     """
     Reads the config map into a dict.
 
     @return: a dictionary representing the version config
     """
-    data = {}
-    with open("../config/versions.txt") as f:
+    with open('../config/versions.txt') as f:
         lines = f.read().splitlines()
-        for line in lines:
-            key, value = line.split("=")
-            data[key] = value
-    return data
+        return dict(line.split('=') for line in lines)
+
 
 def get_build_version():
     """
-    Get the package version from the respective environment parameter, if available.
-    Otherwise, return '0.0.0'.
+    Get the package and development release version from the respective environment parameter, if available.
+    Otherwise, return '0.0.0.dev0'.
+    Following version scheme from https://peps.python.org/pep-0440/#version-scheme
 
     @return: package version
     """
-    return os.getenv('build_version', default='0.0.0')
+    package = os.getenv('BASE_PACKAGE_VERSION', default='0.0.0')
+    dev_release = f'dev{os.getenv("GITHUB_RUN_NUMBER", default=0)}'
```

Comment:
rename the variable, it implies the release version
maybe "dev_run_number" or simply "run_number"?

Reply:
https://peps.python.org/pep-0440/#developmental-releases would suggest `dev_release_segment` or similiar ;)

Reply:
renamed to `dev_release_segment`